doc/source/installation.rst

```diff
@@ -9,19 +9,19 @@ Percona provides packages for popular DEB-based and RPM-based distributions:
 
 * Debian 7 ("wheezy")
 * Debian 8 ("jessie")
-* Ubuntu 12.04 LTS (Precise Pangolin)
+* Debian 9 ("stretch")
 * Ubuntu 14.04 LTS (Trusty Tahr)
 * Ubuntu 16.04 LTS (Xenial Xerus)
 * Ubuntu 16.10 (Yakkety Yak)
```

Comment:
we don't support 16.10 anymore,
17.10 (artful) and 18.04 (bionic) are missing from this list